src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,53 @@
+package core.basesyntax;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import core.basesyntax.service.DataConverter;
+import core.basesyntax.service.ReaderFromCsv;
+import core.basesyntax.service.ReportGenerator;
+import core.basesyntax.service.ShopService;
+import core.basesyntax.service.WriterToCsv;
+import core.basesyntax.service.impl.DataConverterImpl;
+import core.basesyntax.service.impl.ReaderFromCsvImpl;
+import core.basesyntax.service.impl.ReportGeneratorImpl;
+import core.basesyntax.service.impl.ShopServiceImpl;
+import core.basesyntax.service.impl.WriterToCsvImpl;
+import core.basesyntax.strategy.OperationHandler;
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.strategy.impl.BalanceOperationImpl;
+import core.basesyntax.strategy.impl.OperationStrategyImpl;
+import core.basesyntax.strategy.impl.PurchaseOperationImpl;
+import core.basesyntax.strategy.impl.ReturnOperationImpl;
+import core.basesyntax.strategy.impl.SupplyOperationImpl;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    private static final String READ_FILE_PATH = "src/main/resources/ReportToRead.csv";
```

Comment:
Avoid hardcoding file paths in your service classes. It's better to pass the file paths as arguments to your services or use them in the Main class only.